Sources/UIViewAPI/UIViewAPI.swift

```diff
@@ -20,4 +20,24 @@ public extension UIView {
     }
 }
 
+public extension UILabel {
+    var getIntrinsicLabelWidth: CGFloat {
+        get { self.intrinsicContentSize.width }
+    }
+    
+    var getLabelWidth: CGFloat {
+        get { self.frame.width }
+    }
+    
+    func numberOfLines() -> Int {
+      guard let myText = self.text as NSString? else {
+        return 0
+      }
+      // Call self.layoutIfNeeded() if your view uses auto layout
```

Comment:
maybe that should be in the documentation for this method/property? :)

Reply:
Added too :)